app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -176,6 +181,8 @@ def parse_image(image, is_public)
     name ||= $1 if location =~ /^(.+?)(\.(image|img))?\.manifest\.xml$/
     name ||= uid
 
+    labels = parse_labels(image.tags)
```

Comment:
You're storing to custom_attributes with "section" = "labels".
IIUC the real AWS term is "tags" (right?), so perhaps you'd prefer another section name?

(everything about ContainerLabelTagMapping could use a rename, I understand you don't want big changes now, but perhaps naming the section right is better done early.  I'm pretty sure ContainerLabelTagMapping code doesn't care what section you use.)

And whatever the 'section' perhaps the `parse_labels` function should be called `parse_tags`.  Is `parse_*` convention to follow incoming API terminology or how it'll end up in DB terminology?